templates/zabbix-agent-systemd.init.erb

```diff
@@ -1,16 +1,17 @@
 [Unit]
 Description=Zabbix Agent
-After=syslog.target network.target
-Documentation=man:zabbix_agentd
+After=syslog.target
+After=network.target
 
 [Service]
+EnvironmentFile=-/etc/sysconfig/zabbix-agent
```

Comment:
this is redhat specific, but we use the unit file also on other systems:
https://github.com/voxpupuli/puppet-zabbix/blob/master/manifests/startup.pp#L52